src/plugins/data/common/query/text_based_query_state_to_ast.ts

```diff
@@ -57,18 +57,16 @@ export function textBasedQueryStateToExpressionAst({
 
   if (query && isOfAggregateQueryType(query)) {
     const mode = getAggregateQueryMode(query);
-    for (const esMode of ['sql', 'esql']) {
-      if (mode === esMode && esMode in query) {
-        const essql = aggregateQueryToAst({
-          query,
-          timeField: timeFieldName,
-          titleForInspector,
-          descriptionForInspector,
-        });
+    if (mode === 'esql') {
```

Comment:
Nit: I don't think this check is needed anymore since `isOfAggregateQueryType` always means ES|QL now.

Reply:
[7dcf660](https://github.com/elastic/kibana/pull/182079/commits/7dcf660596ac845511abebc8adda264c6ead53d4)